src/app/public/modules/link-list/link-list.component.spec.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyAppTestModule
+} from '@skyux-sdk/builder/runtime/testing/browser';
+
+import {
+  expect
+} from '@skyux-sdk/testing';
+
+import {
+  SkyLinkListComponent
+} from './link-list.component';
+
+describe('Link list component', () => {
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [SkyAppTestModule]
+    });
+  });
+
+  it('should show links', async () => {
+    const fixture = TestBed.createComponent(SkyLinkListComponent);
```

Comment:
We often create the component in the `beforeEach` and assign it to a variable that is scoped for all tests.